src/utils/appLinking.js

```diff
@@ -0,0 +1,119 @@
+ * @example Call the function like this formatUrl[name of the app](inquiry, intent)
+ */
+
+const formatUrl = {
+	email: (inquiry) => (inquiry) = ({ uri: `mailto:${inquiry}`, url: `mailto:${inquiry}` }),
+
+	facebook: (inquiry, intent) => (intent == "groups" && inquiry) ? ({ uri: `https://facebook.com/groups/${inquiry}`, url: `https://www.facebook.com/groups/${inquiry}/` }) :
+		(intent == "profile" && inquiry) ? ({ uri: `https://facebook.com/${inquiry}`, url: `https://www.facebook.com/${inquiry}` }) : ({ uri: "fb://", url: "https://www.facebook.com/" }),
+
+	instagram: (inquiry) => (inquiry) ? ({ uri: `instagram://user?username=${inquiry}`, url: `https://www.instagram.com/_u/${inquiry}/` }) :
+		({ uri: "instagram://", url: "https://www.instagram.com" }),
+
+	linkedin: (inquiry, intent) => (intent === "profile" && inquiry) ? ({ uri: `linkedin://in/${inquiry}`, url: `https://www.linkedin.com/in/${inquiry}` }) :
+		(intent === "company" && inquiry) ? ({ uri: `linkedin://company/${inquiry}`, url: `https://www.linkedin.com/company/${inquiry}` }) :
+			({ uri: "linkedin://", url: "https://www.linkedin.com" }),
+
+	slack: (inquiry, intent) => (intent === "channel" && (inquiry = inquiry in slackInfo.channel ? slackInfo.channel[inquiry] : inquiry)) ?
+		({ uri: `slack://channel?team=${slackInfo.team}&id=${inquiry}` }) : ({ uri: `slack://open?team=${slackInfo.team}` }),
+
+	phone: (inquiry) => (inquiry) = ({ uri: `tel:+${inquiry}`, url: `tel:+${inquiry}` }),
+
+	web: (inquiry) => (inquiry) = ({ uri: `${inquiry}`, url: `${inquiry}` })
+};
+
+/**
+ * @summary
+ * @description This function handles the action launching an app, like a mail app, from our app.
+ * It uses uri schemes and universal liks to do deep linking in both platforms.
+ *
+ *  ** It requires three parameters and an optional warning message. **
+ *  Look under func call to see examples of how to call the function for different apps.
+ *  When dealing with slack,  you can write the name of the channel or if not sure of the name import slackInfo to see the channels available.
+ *
+ *      @param {("email"|"facebook"|
+ * 				 "instagram"|"linkedin"
+ *               |"slack"|"phone"|"web")}  	appName         	The app name to be open.
+ *      @param {("groups"|"profile"|
+ * 				 "company"|"channel"|
+ * 			     "open")} 				    intent        		The intent of the search.
+ *      @param {("announcements"|
+ * 				 "fundraisingCommitte"|
+ * 				 "general"|"motorshpe"|
+ * 				 "projects"|"shpejr")} 		inquiry   			The value for the query search. Slack channels are shown here for easiness.
+ *      @param {String=} 				    warningMessage		Optional value for the alert. It use to populate the alert dialog box.
+ *
+ * @example
+ *  func call
+ *      slack:
+ *              To open the chapter worspace
+ * 			    openAppOrWebsite({"slack"}, {"open"}, {"team" | slackInfo.team}, "Optional error  msg");
+ *
+ *              to send the user directly to a channel, let's say general
```

Comment:
Capitalize first letter of every sentence; change to "To sent the user directly to a channel, e.g. general:"